user_home/models.py

```diff
@@ -0,0 +1,18 @@
+from django.db import models
+
+from user_auth.models import User
+
+
+class Task(models.Model):
+    user_id = models.ForeignKey(User, on_delete=models.CASCADE)
```

Comment:
this should be called user, not user_id
task.user is an object, that contains id
task.user.id